### DISCUSSION TITLE
İbax.io Twitter görevleri neden onaylanmıyor

### DISCUSSION THREAD

**Discussion by cihattheanswer** [IBAX-io/go-ibax#879] [Rewards]
Twitter görevini yapmama rağmen 3.000 ıbxc puanımı vermedi.

**Comment by Skips1** [comment]
Daha önce böyle bi sorun görmedim Düzelte bildinmi ?

**Reply by cihattheanswer** [reply]
Bildiriyim dediğin gibi

**Reply by Eymenedip** [reply]
Yok

**Comment by kublaycelik** [comment]
Hem mobil hemde tarayıcıdan hiç böyle bir sorunla karşılaşmadım. Bazen geç alıyor kafası sistemin ama puan geliyor sonunda
